### PR TITLE
Add Select Subscriptions button, fixes #351

### LINKED ISSUE #351: Make selecting subscriptions more discoverable
See https://github.com/Microsoft/vscode-cosmosdb/issues/351

### PR DESCRIPTION
**Pull request by StephenWeatherford**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,6 +59,20 @@ Check out this [deployment tutorial](https://code.visualstudio.com/tutorials/app
...
+
+If you are not signed in to Azure, you will see a "Sign in to Azure..." link. Alternatively, you can select "View->Command Palette" in the VS Code menu, and search for "Azure: Sign In".
+
+![Sign in to Azure](resources/SignInScreenshot.png)
+
+If you don't have an Azure Account, you can sign up for one today for free and receive $200 in credits by selecting "View->Command Palette" and searching for "Azure: Create an Account".
+
+You may sign out of Azure by selecting "View->Command Palette" and searching for "Azure: Sign Out".
+
+To select which subscriptions show up in the extension's explorer, click on the "Select Subscriptions..." button on any subscription node (indicated by a "key" icon when you hover over it), or select "View->Command Palette" and search for "Azure: Select Subscriptions". Note that this selection affects all VS Code extensions that support the [Azure Account and Sign-In](https://github.com/Microsoft/vscode-azure-account) extension.
```
Comment (EricJizbaMSFT) [inline README.md]: Its not indicated by a "key" icon - its indicated by a "filter" icon. You'll have to update the screenshot, too

File: src/extension.ts [hunk 2]
```diff
@@ -69,6 +69,7 @@ export function activate(context: vscode.ExtensionContext): void {
 
     const actionHandler: AzureActionHandler = new AzureActionHandler(context, outputChannel, reporter);
     actionHandler.registerCommand('appService.Refresh', async (node?: IAzureNode) => await tree.refresh(node));
+    vscode.commands.registerCommand('azureStorage.selectSubscriptions', () => vscode.commands.executeCommand("azure-account.selectSubscriptions"));
```
Comment (EricJizbaMSFT) [inline src/extension.ts]: appService.selectSubscriptions 

And ditto what Nathan said - you have to add this to package.json

**Comment by StephenWeatherford** [comment]
package.json changes got lost, and select subscriptions command was wrong.  Updated README (good catches) and screenshots.

File: src/extension.ts [hunk 3]
```diff
@@ -71,6 +71,7 @@ export function activate(context: vscode.ExtensionContext): void {
 
     const actionHandler: AzureActionHandler = new AzureActionHandler(context, outputChannel, reporter);
     actionHandler.registerCommand('appService.Refresh', async (node?: IAzureNode) => await tree.refresh(node));
+    vscode.commands.registerCommand('appService.selectSubscriptions', () => vscode.commands.executeCommand("azure-account.selectSubscriptions"));
```
Comment (EricJizbaMSFT) [inline src/extension.ts]: Are you intentionally avoiding the actionHandler because you don't want telemetry on this?

Comment (StephenWeatherford) [reply src/extension.ts]: No, hadn't intended that. Good point, I'll change.

**Pull request merged**